sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py

```diff
@@ -385,8 +385,8 @@ class HealthcareEntity(DictMixin):
         :ivar int offset: The Healthcare entity text offset from the start of the document.
         :ivar float confidence_score: Confidence score between 0 and 1 of the extracted
             entity.
-        :ivar links: A collection of entity references in known data sources.
-        :vartype links: list[~azure.ai.textanalytics.HealthcareEntityLink]
+        :ivar data_sources: A collection of entity references in known data sources.
```

Comment:
are you adding the `related_entities` attribute in this pr?

Reply:
I hadn't planned to implement the same graph structure here because I'm not sure how it will be used, and we are also expecting big changes to the relations structure in the next release.  I'd like to defer it for later until we have a chance to review in the context of the upcoming changes in the next API version.

For now I still have `relations` represented as they were previously.

Reply:
ok, cc @mssfang @deyaaeldeen @maririos just for confirmation

Reply:
Java, JS, and .NET are all resolving the relationships and adding them into the HealthcareEntity class like shown [here](https://apiview.dev/Assemblies/Review/b196a5e53fa2441f92e3b1accefa35e7/7a57229e63bd42fd8e7878b1883cbec1?doc=False#Azure.AI.TextAnalytics.HealthcareEntity).

If Python wants to wait I guess that is ok. For the UX study though, we need to have this decided and implemented.

Reply:
@abhahn I think this is a fairly big divergence from other languages which I think should have been brought up in the scrum at least. I would suggest to reach out to @johanste and Mayuri to get approval first if you did not already.

Reply:
If it will be a major issue I will do my best to have it implemented tomorrow, but it may not be finished until the end of the day.  I am mainly concerned about making it in time for code complete, especially since there are timezone differences for reviewers.

Reply:
No worries, If you need one person to review it on the weekend, I can review it.

Reply:
@iscai-msft , @maririos , @mssfang , @deyaaeldeen , I've added `related_entities` to `HealthcareEntity` and removed `relations` from `AnalyzeHealthcareEntitiesResultItem`.  It took less time than I thought it would.  I still need to add tests, but have a look at the samples for now and let me know what you think.

Reply:
Thanks @abhahn! Looks good!